dbqueries.py

```diff
@@ -34,18 +34,20 @@ class TableRecordCountsQueryData(QueryData):
 
 class QueryDict(TypedDict):
     udp_context_store_view_counts: TableRecordCountsQueryData
+    udp_learner_activity_updated: StandardQueryData
 
 
 QueryName = Literal[
-    'udp_context_store_view_counts'
+    'udp_context_store_view_counts',
+    'udp_learner_activity_updated'
 ]
 
 
 # Check functions
 
 NOT_ZERO: Callable[[int], bool] = (lambda x: x != 0)
 LESS_THAN_TWO: Callable[[int], bool] = (lambda x: x < 2)
-LESS_THAN_TWO_DAYS: Callable[[str], bool] = (lambda x: (datetime.now(tz=timezone.utc) - datetime.fromisoformat(x)).days < 2)
+LESS_THAN_TWO_DAYS: Callable[[str], bool] = (lambda x: (datetime.now(tz=timezone.utc) - x.replace(tzinfo=timezone.utc)).days < 2)
```

Comment:
I modified this since it seems the updated_date field is no longer being given as a iso formatted string, instead its a datetime object without tzinfo